uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
         }
         return render('summary/food_eligible.xml', {'attendees': eligible})
 
+    def csv_import(self, message='', all_instances=None, form_link=None):
+
+        return {
+            'message': message,
+            'tables': sorted([model.__name__.lower() for model in Session.all_models()]),
```

Comment:
Roger. Although, I expect that the select box will then display something like "<class='Attendee' blah blah>"? Is there a way to spit out a tuple list that displays a nice name with the value being the model?